### ISSUE TITLE
[Info] Run HTTP on Linux 

### ISSUE THREAD

**Issue by WoofHax** [GrowtopiaNoobs/GrowtopiaServer#1083]
if you guys Trying to run server HTTP on Linux maybe my repositories can help a bit 

https://github.com/WoofHax/Launcher-Server-Growtopia

**Comment by NotDark** [comment]
sudo apt-get install apache2

**Comment by kingawesomed2** [comment]
Retard

**Comment by atomiclucky** [comment]
yeah lmao. running a http server on linux is even easier than on windows (Unless you don't know how to use the terminal)

**Comment by WoofHax** [comment]
But running http for gt server

**Comment by playingo** [comment]
xampp sucks use nginx

**Comment by atomiclucky** [comment]
dude you just copied the code from growtopianoobs wiki and added your name into it

**Comment by WoofHax** [comment]
@ardakurucim I Can't believe that you didn't read my README.md nice

**Comment by Cojeb** [comment]
You can do this in multiple languages including nodejs, python and more.

**Comment by Cornysz** [comment]
ok so can i run gt priv server with vps linux with this? am i wrong?
